app/views/admin/grove_monitor/index.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="grove-monitor text-center">
+  <h2>Grove Monitor</h2>
+</div>
+
+<ul class="nav nav-pills">
+  <li role="presentation" class="active"><%= link_to "All", admin_grove_monitor_all_path %></li>
+  <li role="presentation" class="active"><%= link_to "Absent", admin_grove_monitor_path(filter: :absent) %></li>
+  <li role="presentation" class="active"><%= link_to "Lost", admin_grove_monitor_path(filter: :lost) %></li>
+  <% @presenter.locations.keys.each do |location| %>
+    <li role="presentation" class="active"><%= link_to location, admin_grove_monitor_path({filter: :location, name: location}) %></li>
+  <% end %>
+</ul>
+
+<div class="container">
+  <div class="row">
+
+  <h2>Lost</h2>
+
+  <% @presenter.lost_students.each do |student| %>
+      <div class="col-xs-3">
+        <div class="thumbnail">
+          <img src="..." alt="...">
+          <div class="caption">
+            <h3><%= student.name %></h3>
+            <p>...</p>
+            <p><a href="#" class="btn btn-primary" role="button">Button</a> <a href="#" class="btn btn-default" role="button">Button</a></p>
+          </div>
+        </div>
+      </div>
+  <% end %>
+
+  <h2>Absent</h2>
+
+  <% @presenter.absent_students.each do |student| %>
```

Comment:
It looks like the student presentation is the same in each of these blocks. It would be easier to use a partial with collection:
http://guides.rubyonrails.org/layouts_and_rendering.html
for example:
`<%= render partial: "student", collection: @presenter.absent_students %>`